src/storhaug

```diff
@@ -301,10 +301,11 @@ create_virt_ip()
     pcs -f ${cibfile} constraint location vip${ipcount} rule resource-discovery=exclusive score=0 role eq storage
 
     pcs -f ${cibfile} resource create vip${ipcount}_trigger ocf:heartbeat:ganesha_trigger \
-        meta resource-stickiness="0"
+        --clone vip${ipcount}_trigger-clone vip${ipcount}_trigger \
+	meta interleave="true" globally-unique="false" notify="true" clone-max="${STORAGE_NUM_SERVERS}"
```

Comment:
I think I understand most of what you're doing, but why is `clone-max` not 1? Shouldn't there ever only be one, the one that follows the `vip${ipcount}` resource?